vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java

```diff
@@ -174,7 +174,7 @@ public void testSimpleQuestionAnswer() throws Exception {
 
   @Test
   public void testQuickstart() throws IOException {
-    String output = Quickstart.quickstart(PROJECT_ID, LOCATION, GEMINI_PRO_VISION);
+    String output = Quickstart.quickstart(PROJECT_ID, LOCATION, GEMINI_FLASH);
```

Comment:
Should the same change be made for `testTextInput()` in line 203?

Reply:
Good catch! Added for testTextInput().